migrations/20200304-add-column_groupEventId_to_events.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable new-cap */
+
+'use strict';
+
+const prefix = process.env.DATASTORE_SEQUELIZE_PREFIX || '';
+const table = `${prefix}builds`;
+
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        await queryInterface.sequelize.transaction(async (transaction) => {
+            await queryInterface.addColumn(table, 'templateId', {
```

Comment:
shouldn't this be `groupEventId`

Reply:
renamed the file and github uploaded the wrong one
deleted the other one and the changes are reflected correctly now